### ISSUE TITLE
Deployment on server

### ISSUE THREAD

**Issue by racerlucas** [btx0424/OmniDrones#61]
In our lab, we don't have workstation for researcher, so is there any way to deploy the whole environment on a headless server?

**Comment by btx0424** [comment]
Yes. You can install Isaac Sim locally on a PC and then copy the root directory (which is by default `${HOME}/.local/share/ov/pkg/isaac_sim-*`) to the server. Then set `$ISAACSIM_PATH` on the server.

However, you'd better have sudo on the server in case you encounter driver issues or [file change watcher limit](https://docs.omniverse.nvidia.com/dev-guide/latest/linux-troubleshooting.html?highlight=watch#q11-getting-many-failures-similar-to-failed-to-create-change-watch-for-xxx-errno-28-no-space-left-on-device).

**Comment by racerlucas** [comment]
I've done the container deployment provided by the Isaac Sim documents, should redo everything as you say or use the container? Because I think I still need the video.

**Comment by btx0424** [comment]
I think the container option is fine. You do not have to redo as I said. The container installation should be able to produce videos as well.

It might be slower, though. If you are unsatisfied with the performance, try what I said.

**Comment by racerlucas** [comment]
So how to do the $ISAACSIM_PATH thing after installing the container?

**Comment by btx0424** [comment]
You can first locate the root directory of Isaac Sim (should be under /isaac_sim if I remembered correctly) and export it as an environment variable in ~/.bashrc Sent from my iPhoneOn 22 Mar 2024, at 3:03 PM, racerlucas ***@***.***> wrote:﻿
So how to do the $ISAACSIM_PATH thing after installing the container?

**Comment by racerlucas** [comment]
I tried, but then the only thing left in root directory it the kit dir., that's quite weird。
